contracts/rewards/RewardsMath.sol

```diff
@@ -25,14 +22,18 @@ library RewardsMath {
 
     /**
      * @dev returns the amount of rewards distributed after a given time period since deployment has elapsed
-     * The returned value is calculated as `totalRewards * (1 - 1 / e ^ (timeElapsed * LAMBDA))`.
-     * Note that because the exponentiation function is limited to an input of up to (and excluding) 16, the
-     * input value to this function is limited by `timeElapsed * LAMBDA < 16` --> `timeElapsed < 1120000000`.
-     * For `timeElapsed = 1120000000 - 1`, the formula above returns more than 99.9999% of `totalRewards`.
+     *
+     * The returned value is calculated as `totalRewards * (1 - 1 / 2 ^ (timeElapsed / halfLife))`
```

Comment:
(Comments should start with lowercase letters)

Reply:
done